style2.css

```diff
@@ -147,4 +405,83 @@ img {
     flex-direction: column;
     align-items: flex-start;
   }
+  .nav {
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    height: 120px;
+  }
+
+  .market-open {
+    top: -25px;
+    left: 250px;
+  }
+
+  .bell,
+  .bell-text {
+    margin: 0 auto;
+  }
+  .bell-text {
+    top: -40px;
+  }
+  li {
+    display: block;
+  }
+  #saved-alerts-container {
+    background-color: #f1edec;
+    height: fit-content;
+    width: 100%;
+    position: absolute;
+    top: 1150px;
```

Comment:
Notice that screens can be of different size, both in terms of width and height. For example, if the user does not open the window maximized and with a height less than 1150px, will they not be able to see the saved alerts container as a result?

Reply:
ohhh, i will check that out!